rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :person, :book
+
+  def initialize(date)
```

Comment:
- [ ] The constructor of Rental should be modified so Book and Person are set in it to break the many-to-many relationship. e.g
`def initialize(date, book, person)`
You can read more about this [here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)